docs/user_docs/overview/supported-addons.md

```diff
@@ -8,107 +8,217 @@ sidebar_label: Supported addons
+
+### MySQL and its variants
+
+**Addon List**
+
+| Addons          | Description                                                                                                                                                                                                                                                                                                                                                             |
+|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| mysql           | This addon uses the community edition MySQL image officially released by Oracle.                                                                                                                                                                                                                                                                                        |
+| apecloud-mysql  | ApeCloud MySQL is a free, fully compatible drop-in replacement for MySQL Community Edition, offering enhanced high availability through a RAFT protocol replication plugin. The image is provided by ApeCloud. Additionally, ApeCloud MySQL includes an open-source proxy called WeScale, which provides features such as read-write splitting and connection pooling.  |
+| mariadb         | MariaDB is a high performance open source relational database management system that is widely used for web and application servers.                                                                                                                                                                                                                                    |
+**Supported Features**
 
 :::note
 
-- The versions listed below may not be up-to-date, and some supported versions might be missing. For the latest addon versions, please refer to the [KubeBlocks addon GitHub repo](https://github.com/apecloud/kubeblocks-addons).
-- The upgrade feature means that KubeBlocks supports minor version upgrades for a database engine. For example, you can upgrade PostgreSQL from v12.14 to v12.15.
+The versions listed below may not be up-to-date, and some supported versions might be missing. For the latest addon versions, please refer to the [KubeBlocks addon GitHub repo](https://github.com/apecloud/kubeblocks-addons).
 
 :::
 
 | Addon (v0.9.0)     | version                                                    | Vscale           | Hscale           | Volumeexpand | Stop/Start       | Restart          | Expose | Backup/Restore | Logs | Config | Upgrade (DB engine version) | Account | Failover | Switchover | Monitor |
```

Comment:
Replace "version" in the table with "Version" or "Supported Version" throughout this tutorial.

Reply:
Seems should be ”Supported Versions“